packages/login/test/login-form-wrapper-styles.js

```diff
@@ -0,0 +1,12 @@
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+registerStyles(
+  'vaadin-login-form-wrapper',
```

Comment:
Extracted CSS to separate file because otherwise `registerStyles` is calleld when the class is already finalized, and the corresponding `ThemableMixin` warning is shown.